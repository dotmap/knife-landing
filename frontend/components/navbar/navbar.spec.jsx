/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import NavBar from '.'

describe('NavBar component', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<NavBar />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
