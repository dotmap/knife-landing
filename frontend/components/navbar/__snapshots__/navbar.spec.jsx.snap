// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component matches snapshot 1`] = `
<nav
  className="css-1ph70be"
>
  <div
    className="css-e6ghfl"
    width={0.3333333333333333}
  >
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "black",
          "textDecoration": "none",
        }
      }
    >
      <h2
        className="css-esj6bp"
        fontFamily="inter"
        fontSize={4}
        fontWeight="bold"
      >
        knife
      </h2>
    </a>
  </div>
  <div
    className="css-1cw4iel"
    width={0.6666666666666666}
  >
    <a>
      <button
        className="css-1uifvfx"
        color="black"
        fontSize="inherit"
        fontWeight="bold"
      >
        <div
          className="css-4pml4y"
          fontFamily="inter"
        >
          Sign In
        </div>
      </button>
    </a>
  </div>
</nav>
`;
